doc/chapters/api/index.rst

```diff
@@ -39,11 +46,16 @@ created.
 
 Parameters
 ##########
-* containerID: ``int32`` The ID obtained by CreateContainer method.
+* containerID: ``int32`` The ID obtained by Create method.
 * pathInHost: ``string`` absolute path in the host.
 * pathInContainer: ``string`` the absolute path to mount to in the container.
 * readOnly: ``bool`` indicates whether the directory is read-only or not.
 
+Prerequisities
+##############
+* A container that has been successfully created, such that the Create call returned a containerID.
```

Comment:
i would like more concise wording like : "Create method called and containerID is gathered"